run-sample-distribution.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
We should move this to within `analyses/sample-distribution-analyses` per @cgreene's point.